src/frontend/pages/signup/signup.jsx

```diff
@@ -1,7 +1,55 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { Navbar } from "../../components/navbar/Navbar";
+import "../login/auth.css";
 
 export function Signup(){
     return(
-        <h1> signup page </h1>
+       <>
+       <Navbar />
+        <div className="login-card flex-col">
+            <h1 className="text-center margin-zero">Sign up</h1>
+            <form className="flex-col">
+                <div className="flex-col-login">
+                    <label className="larger-font" for="email">Email address</label>
+                    <input className="input padding-top-12px" type="email" placeholder="abc@gmail.com" required/>
+                </div>
+
+                <div className="flex-col">
+                    <label className="larger-font" for="password">Password</label>
+                    <input className="input padding-top-12px" placeholder="&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;" id='password' type="password" required/>
+                </div>
+
+                <div className="flex-col-login">
+                    <label className="larger-font" for="password">Re-type Password</label>
```

Comment:
in your screenshot these seems to be an odd amount of padding around here , do check and inspect why that happens and make the paddings between labels and inputs uniforms. 
Not mandatory, but good to do .

Reply:
yes padding has been fixed now